app/events/db/db_table_field_mappings.yaml

```diff
@@ -906,3 +906,149 @@ mappings:
                 logo_url: work_platform_logo_url
             profile_url: profile_url
             content_url: content_url
+
+
+    profiles_search_export_data:
+        name: profiles_search_export_data
+        schema: iiq_schema
+        unique_key: iiq_id
+        fields:
+            job_id: profiles_search_export_request_id
+            work_platform:
+                id: work_platform_id
+                name: work_platform_name
+                logo_url: work_platform_logo_url
+            platform_username: platform_username
+            url: url
+            image_url: image_url
+            full_name: full_name
+            introduction: introduction
+            is_verified: is_verified
+            platform_account_type: platform_account_type
+            gender: gender
+            age_group: age_group
+            language: language
+            follower_count: follower_count
+            subscriber_count: subscriber_count
+            content_count: content_count
+            engagement_rate: engagement_rate
+            location:
+                city: city
+                state: state
+                country: country
+            contact_details:
+                type: contact_type
+                value: contact_value
+
+
+    profiles_search_export_request:
+        name: profiles_search_export_request
+        schema: iiq_schema
+        unique_key: iiq_id
+        fields:
+            id: iiq_id
+            status: status
+            work_platform:
+                id: work_platform_id
+                name: work_platform_name
+                logo_url: work_platform_logo_url
+            follower_count_min: follower_count_min
+            follower_count_max: follower_count_max
+            subscriber_count_min: subscriber_count_min
+            subscriber_count_max: subscriber_count_max
+            content_count_min: content_count_min
+            content_count_max: content_count_max
+            audience_gender_type: audience_gender_type
```

Comment:
since these filters are too many, and the chance of new ones getting added is high, and these are just for reference, they won't be used programatically anywhere after the request has been made, can we dump them all in a single jsonb column?

@kushal-o2 what do you think?

Reply:
Discussed. Will pick this later in a separate issue.